cpp/include/cuspatial/experimental/detail/is_point_in_polygon_kernel.cuh

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Suggest removing the name "kernel" from this file since it doesn't contain a kernel.

Reply:
Done.